HW12_3.py

```diff
@@ -0,0 +1,12 @@
+from datetime import datetime
+
+class MyCustomException (Exception):
+    start_time=datetime.now()
```

Comment:
It is better to move this part inside of the __init__ method to have logs whet the exception is raised but not initialized:
```
class MyCustomException(Exception):
    def __init__(self, message):
        # write to a file here

Reply:
done